ext/node/polyfills/testing.ts

```diff
@@ -22,6 +56,10 @@ class NodeTestContext {
     this.#denoContext = t;
   }
 
+  get assert() {
+    return getAssertObject();
+  }
```

Comment:
Q: Why do we need to clone the assert object in the first place? Does Node only expose a subset here? Otherwise we could skip the object creation:

```ts
  get assert() {
    return assert
  }
```

Reply:
Yes, only a subset of properties is exposed.